### PR TITLE
update readme.md. 

### LINKED ISSUE #972: update readme working mode
The readme now still not using `kernel-native` and `dual-enginee `

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #972 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,18 +30,18 @@ The main components of Kmesh include:
 - **eBPF Orchestration**: The traffic orchestration implemented based on eBPF, supports L4 load balancing, traffic encryption, monitoring and simple L7 dynamic routing.
 - **Waypoint**: Responsible for advanced L7 traffic governance, can be deployed separately per namespace, per service.
 
-Kmesh innovatively sinks Layer 4 and Simple Layer 7 (HTTP) traffic governance to the kernel, and build a transparent sidecarless service mesh without passing through the proxy layer on the data path.
+Kmesh innovatively sinks Layer 4 and Simple Layer 7 (HTTP) traffic governance to the kernel, and build a transparent sidecarless service mesh without passing through the proxy layer on the data path. We named this Kernel-native-mode.
```
Comment (YaoZengzeng) [inline README.md:33]: The content of picture `docs/pics/kmesh-arch.png` also need to be updated.

Comment (hzxuzhonghu) [inline README.md:33]:
```suggestion
Kmesh innovatively sinks Layer 4 and Simple Layer 7 (HTTP) traffic governance to the kernel, and build a transparent sidecarless service mesh without passing through the proxy layer on the data path. We named this Kernel-Native mode.
```

File: README.md [hunk 2]
```diff
@@ -30,18 +30,18 @@ The main components of Kmesh include:
...
 
 <div align="center">
-    <img src="docs/pics/simple-mode.png" alt="simple-mode" width="800" />
```
Comment (hzxuzhonghu) [inline README.md:36]: should we remove the image?

Comment (LiZhenCheng9527) [reply README.md:36]: Can be removed

Comment (LiZhenCheng9527) [reply README.md:36]: no. I had renamed it. 
Why remove it?

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**